src/umpire/ResourceManager.cpp

```diff
@@ -225,6 +230,21 @@ ResourceManager::initialize()
     m_allocators.emplace_front(std::move(allocator));
   }
 
+  {
```

Comment:
I think we need to consider guarding this with `#if defined(UMPIRE_ENABLE_FILE)`